docs/general/polkadot-v2.md

```diff
@@ -0,0 +1,234 @@
+---
+id: polkadot-v2
+title: Potential Changes to Polkadot
```

Comment:
I don't think this is a "potential change", it is more like the next iteration of how we're viewing Polkadot; a perspective change, or a paradigm shift if you may.

Reply:
changed to Polkadot Direction